extension/ts/workers/invoker.ts

```diff
@@ -1,6 +1,6 @@
 import {WorkerKind, WorkerRequest, WorkerResponse} from "./types";
 
-const invokeWorker = async <K extends WorkerKind>(kind: K, request: WorkerRequest<K>): Promise<WorkerResponse<K>> => {
+const invokeWorker = <K extends WorkerKind>(kind: K, request: WorkerRequest<K>): Promise<WorkerResponse<K>> => {
```

Comment:
uhhh why can we do this
is it unnecessary because it returns a promise?

Reply:
Yeah you _only_ need the `async` when you're `await`ing or wrapping synchronous code in a Promise.

This at best did nothing, and at worst added a layer of calls on the event loop